### PR TITLE
div. Bugs

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/views/templates/cpa/_person_details.gsp [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
         <div class="ui item person-details">
             <h5 class="ui header">
-                <g:link controller="person" action="show" id="${person?.id}">
+
                     ${person?.title}
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/cpa/_person_details.gsp:7]: person?

File: app/grails-app/views/templates/cpa/_person_details.gsp [hunk 2]
```diff
@@ -49,12 +49,12 @@
         <div class="ui item person-details">
             <div class="content la-space-right">
                 <h5 class="ui header">
-                    <g:link controller="person" action="show" id="${personRole?.prs?.id}">
+
                         ${personRole?.prs?.title}
```
Comment (croqueGrec09) [inline app/grails-app/views/templates/cpa/_person_details.gsp:53]: personRole?.prs?

File: app/grails-app/views/user/_tableDueDates.gsp [hunk 3]
```diff
@@ -65,7 +65,7 @@
                         <g:elseif test="${obj instanceof de.laser.base.AbstractPropertyWithCalculatedLastUpdated}">
                             <g:if test="${obj.owner instanceof Person}">
                                 <i class="icon address book la-list-icon"></i>
-                                <g:link controller="person" action="show" id="${obj.owner.id}">${obj.owner?.first_name}&nbsp;${obj.owner?.last_name}</g:link>
+                                <${obj.owner?.first_name}&nbsp;${obj.owner?.last_name}
```
Comment (croqueGrec09) [inline app/grails-app/views/user/_tableDueDates.gsp:68]: instance check should eliminate null, null cannot be instanceof Person

**Review by croqueGrec09 — COMMENTED** [review]
some unnecessary null checks

File: app/grails-app/views/user/_tableDueDates.gsp (hunk 3 above)
Comment (klodav) [inline app/grails-app/views/user/_tableDueDates.gsp:68]: obj.owner exists here -> ?

File: app/grails-app/services/de/laser/CopyElementsService.groovy [hunk 4]
```diff
@@ -420,7 +420,9 @@ class CopyElementsService {
         if (formService.validateToken(params)) {
 
             params.list('copyObject.take').each { takeProperty ->
+                println(takeProperty)
```
Comment (klodav) [inline app/grails-app/services/de/laser/CopyElementsService.groovy:423]: here and below several times : println

**Review by croqueGrec09 — APPROVED** [review]
approved

File: app/grails-app/views/templates/cpa/_person_details.gsp [hunk 5]
```diff
@@ -3,27 +3,27 @@
...
+
             </h5>
 
         </div><!-- .person-details -->
 
-        <g:if test="${person?.contacts}">
+        <g:if test="${person.contacts}">
 
-            <g:each in="${person?.contacts?.toSorted()}" var="contact">
+            <g:each in="${person.contacts?.toSorted()}" var="contact">
```
Comment (klodav) [inline app/grails-app/views/templates/cpa/_person_details.gsp:18]: person.contacts?

File: app/grails-app/views/templates/cpa/_person_details.gsp [hunk 6]
```diff
@@ -3,27 +3,27 @@
...
                 <g:render template="/templates/cpa/contact"
                           model="${[contact: contact, tmplShowDeleteButton: tmplShowDeleteButton, overwriteEditable: overwriteEditable]}"/>
             </g:each>
 
         </g:if>
-        <g:if test="${person?.addresses}">
+        <g:if test="${person.addresses}">
 
-            <g:each in="${person?.addresses?.sort { it?.type.each {it?.getI10n('value')} }}" var="address">
+            <g:each in="${person.addresses?.sort { it?.type.each {it?.getI10n('value')} }}" var="address">
```
Comment (klodav) [inline app/grails-app/views/templates/cpa/_person_details.gsp:26]: person.addresses?

File: app/grails-app/views/templates/cpa/_person_details.gsp [hunk 7]
```diff
@@ -89,9 +89,9 @@
                 </g:if>
             </div>
         </div><!-- .person-details -->
-    <g:if test="${personRole?.prs?.contacts}">
-        <g:each in="${personRole?.prs?.contacts?.toSorted()}" var="contact">
-            <g:if test="${tmplConfigShow.contains(contact?.contentType?.value)}">
+    <g:if test="${personRole.prs.contacts}">
+        <g:each in="${personRole.prs.contacts?.toSorted()}" var="contact">
```
Comment (klodav) [inline app/grails-app/views/templates/cpa/_person_details.gsp:93]: personRole.prs.contacts?

**Comment by Moetez16** [comment]
Done

**Review by klodav — APPROVED** [review]

**Pull request merged**